<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendRequestByPatch_Dialog</class>
 <widget class="QDialog" name="SendRequestByPatch_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="numRequestByPatchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>需要批量发送的请求数：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="assignPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numRequestByPatchSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
